api4/webhook.go

```diff
@@ -413,20 +437,24 @@ func getOutgoingHook(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	c.LogAudit("attempt")
+	auditRec := c.MakeAuditRecord("getOutgoingHook", audit.Fail)
```

Comment:
Is there a reason why the audit logging has been removed from `getIncomingHook()` but retained for `getOutgoingHook()`?

Reply:
No good reason.  I may have decided that `get` methods don't require auditing and then contradicted myself a hundred lines later.  I added auditing back as it was there before.